homeassistant/components/mikrotik/hub.py

```diff
@@ -82,9 +83,20 @@ def force_dhcp(self) -> bool:
         """Return force_dhcp option setting."""
         return self.config_entry.options.get(CONF_FORCE_DHCP, False)
 
+    def detect_routerboard_router(self) -> bool:
+        """Detect whether or not it's a routerboard router."""
+        return len(self.command(MIKROTIK_SERVICES[INFO])) != 0
+
```

Comment:
Can we update the value of `self._is_routerboard` under the `command` method instead of logging a warning when the `info` command is called?
And you can call this method under `get_hub_details` without returning any value.

Reply:
The method `detect_routerboard_router` is not needed anymore as `command` now considers that it's not a routerboard if there is a "no such command" error with the INFO command.